signer/metrics/classifier/default.go

```diff
@@ -28,6 +28,10 @@ func Default(ctx context.Context, startDag *dag.Dag, endDag *dag.Dag) (classific
 		return
 	}
 
+	if _, ok := dataMap["_product"]; ok {
+		classification = "327dc2e"
```

Comment:
I can totally see this being a serious headscratcher in the not too distant future. I'm not sure what to do so I guess I'm ok with it for now.

Reply:
I added a "See Kibana" note that at least points future us in the right direction.